swift-package-baseline/breakage-allowlist-path.txt

```diff
@@ -1 +1 @@
-
+API breakage: enumelement VisualInstruction.Component.guidanceView has declared type change from (MapboxDirections.VisualInstruction.Component.Type) -> (MapboxDirections.GuidanceViewImageRepresentation, MapboxDirections.VisualInstruction.Component.TextRepresentation) -> MapboxDirections.VisualInstruction.Component to (MapboxDirections.VisualInstruction.Component.Type) -> (MapboxDirections.GuidanceViewImageRepresentation, MapboxDirections.VisualInstruction.Component.TextRepresentation, MapboxDirections.VisualInstruction.Component.GuidanceViewKind?) -> MapboxDirections.VisualInstruction.Component
```

Comment:
I'm wondering why there were no 'breaking changing' comments in this PR #778 
It looks like there is a problem [the breaking changes detection build](https://app.circleci.com/pipelines/github/mapbox/mapbox-directions-swift/1947/workflows/e59a70c6-df93-43e9-852d-4686ee6807ea/jobs/9022). It has build error, but the build status is green.
> error: Couldn’t get revision ‘v2.9.0-rc.2^{commit}’:
    fatal: Needed a single revision